libs/sdk-ui-kit/src/ColorPicker/tests/ColorPicker.test.tsx

```diff
@@ -14,9 +14,9 @@ const initialRgbColor: ColorFormats.RGB = {
     b: 0,
 };
 
-const defaultStyle: React.CSSProperties = {
-    backgroundColor: "hsl(0, 100%, 50%)",
-    borderColor: null,
+const defaultStyle: Record<string, unknown> = {
```

Comment:
Why is this change necessary?

Reply:
`toHaveStyle` method using this variable was from `jest-enzyme` matchers and after removal it was replaced by a same named matched from RTL with different input type

Reply:
should probably be a part of the second commit - I will move it